file-formats/scp1/type/zif_aff_scp1_v1.intf.abap

```diff
@@ -81,10 +81,10 @@ INTERFACE zif_aff_scp1_v1
     BEGIN OF co_operation_at_activation,
       "! <p class="shorttext">Insert/Update</p>
       "! Data records which are to be inserted/updated at activation are flagged with value ''
```

Comment:
I prefer `modify` (instead of `insert_or_update`). Would you update the related ABAP Doc accordingly, please? I am wondering whether the description should contain the value that is stored on the database. I think this is not relevant for ABAP file formats.

```suggestion
      "! <p class="shorttext">Modify</p>
      "! Data records which are to be modified at activation
```

Reply:
Yes, I missed to change the ABAP Doc. I have updated the ABAP Doc.